packages/paste-libraries/styling/src/css-function/__tests__/css.test.ts

```diff
@@ -0,0 +1,480 @@
+import css from '../css';
```

Comment:
Mainly copy pasta from styled system, just a few tests update to meet our theme key requirements for colors